superset/utils/data.py

```diff
@@ -14,17 +14,36 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+import decimal
+import importlib.util
+import json
+import logging
+import os
 import random
+import re
 import string
 import sys
 from datetime import date, datetime, time, timedelta
-from typing import Any, Callable, cast, Dict, List, Optional
+from inspect import getsource
+from pathlib import Path
+from types import ModuleType
+from typing import Any, Callable, cast, Dict, List, Optional, Set, Type
+from uuid import uuid4
 
 import sqlalchemy.sql.sqltypes
+import sqlalchemy_utils
+from flask_appbuilder import Model
+from graphlib import TopologicalSorter  # pylint: disable=wrong-import-order
```

Comment:
Is this available only in Python 3.9?

Reply:
It is, but I added a backport (`graphlib-backport`) as a dependency.